mood-analysis.rb

```diff
@@ -19,6 +21,39 @@ def analyze_mood(words)
   return ":-|"
 end
 
+def strip_punctuation(word)
+  word.gsub!(/[!.,#]/, "") unless word.nil?
+end
+
+
+def happy_days(text)
+  days = 0
+  index = analyze_text(text).each_with_index do |mood, i|
+    days += 1 if mood.include? ":-)"
+    break i if days == 3
+  end
+
+  return "There aren't 3 happy days!" if days < 3
+  "It takes #{index} entries for 3 happy days to occur."
+end
```

Comment:
This method should return a fixnum and you should use the result of the method in a puts statement outside of the method. This allows us to do other things with this information rather than just insert it into this string.